paladin/README.md

```diff
@@ -0,0 +1,238 @@
+# Paladin
+
+Paladin helps secure your Drash applications through headers. Inspired by [helmet](https://github.com/helmetjs/helmet). It is configurable and can be used throughout the request-resource-response lifecycle.
+
+Paladin will set extra HTTP headers to the request response, to aid in securing your application. This does not make your application bulletproof, but adds extra security layers.
+
+```typescript
+import { Drash } from "https://deno.land/x/drash@{version}/mod.ts";
+
+// Import the Paladin middleware function
+import { Paladin } from "https://deno.land/x/drash-middleware@{version}/paladin/mod.ts";
+
+// Instantiate paladin
+const paladin = Paladin();
+
+// Create your server and plug in paladin to the middleware config
+const server = new Drash.Http.Server({
+  resources: [
+    HomeResource,
+  ],
+  middleware: {
+    after_request: [
+      paladin
+    ]
+  }
+});
+
+server.run({
+  hostname: "localhost",
+  port: 1447,
+});
+
+console.log(`Server running at ${server.hostname}:${server.port}`);
+```
+
+## Configuration
+
+Paladin has a list of default headers it will set, when `Paladin` is called with no arguments, but you can override these.
+
+### `X-XSS-Protection`
+
+This header does not protect against XSS attacks, but only protects against *one type* of XSS attack.
+
+See here for more information: https://helmetjs.github.io/docs/xss-filter/
+
+- [x] Enabled by default?
+
+```typescript
+const paladin = Paladin() // Set by default
+const paladin = Paladin({
+  "X-XSS-Protection": false // Disable it
+})
+const paladin = Paladin({
+  "X-XSS-Protection": true // Explicitly enable it
+})
+```
+
+### `Referrer-Policy`
+
+The Referrer-Policy header is usually set by browsers to tell a server where it's coming from.
+
+See here for more information: https://helmetjs.github.io/docs/referrer-policy/
+
+- [ ] Enabled by default?
+
+```typescript
+const paladin = Paladin() // Will not set the header
+const paladin = Paladin({
+  "Referrer-Policy": "origin" // Enable it, and set it to "origin"
+})
+```
+
+See [here](https://www.w3.org/TR/referrer-policy/#referrer-policies) for all possible policies.
+
+### `X-Content-Type-Options`
+
+The Referrer-Policy header is used to tell browsers not to sniff the mime type. Browsers will trust what the server says and block the resource if it’s wrong.
```

Comment:
change mime to [MIME](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)

Reply:
https://github.com/drashland/deno-drash-middleware/pull/25/commits/b8b3bba86166d57696b710bdb3513861ded928cd